### DISCUSSION TITLE
Can this use stack navigation inside the drawer side menu?

### DISCUSSION THREAD

**Discussion by putuoka** [lukebrandonfarrell/react-native-navigation-drawer-extension#97] [Q&A]
I need to use stack navigation inside drawer side menu to change the menu content dynamically. is it possible?

**Comment by lukebrandonfarrell** [comment]
I haven't worked with the library for a while, what have you tried? and what issues are you getting?

**Reply by putuoka** [reply]
> I haven't worked with the library for a while, what have you tried? and what issues are you getting?

sorry my English isn't good i'll try to explain. What i want to achieve is to change text/content inside side menu drawer(in react native navigation drawer it's called as custom "drawercontent") and keep the show only inside the side menu not is a full width screen. so i mean when clicking text in side menu, the change happen in side menu. so to achieve this i need to put stack navigation inside side menu drawer OR the other way is to use hide and show trick(like usestate)
